index.html

```diff
@@ -46,7 +46,7 @@
 				<h2>Country</h2>		
 				<select name="country">
 					<option value selected diasabled>Choose your country</option>
-				  	<option value="AF">Afghanistan</option>
+				  <option value="AF">Afghanistan</option>
```

Comment:
The whitespace prior to the `option` tag uses tab just like the other tags in the HTML doc.